Readme.md

```diff
@@ -142,3 +142,19 @@ proper certs and tokens. Systemd files are used to manage and start the daemons.
 ```bash
 ./nnf-deploy install
 ```
+
+# Testing
+
+NNF test infrastructure and individualized tests reside in the `/test` directory. Tests are expected to run against a fully deployed cluster reachable via your current k8s configuration context. NNF test uses the [Ginkgo](https://onsi.github.io/ginkgo) test framework.
+
+Various Ginkgo options can be passed into `go test`. Common options include `-ginkgo.fail-fast`,  `-ginkgo.progress`,  and `-gingo.v`
+
+```bash
+go test -v ./test/... -ginkgo.fail-fast -ginkgo.progress -ginkgo.v
+```
+
+Ginkgo also provides the [Ginkgo CLI](https://onsi.github.io/ginkgo/#ginkgo-cli-overview) that can be used for enhanced test features like parallelization, randomization, and filter.
+
+## Test Definitions
+
+Individual tests are listed in `/test/int_test.go`. Tests are written from the perspective of a workload manager and should operate only on DWS resources when possible.
```

Comment:
Is that to say that storage/container profiles are not DWS resources?

Reply:
Also, add a newline at the end: MD047/single-trailing-newline: Files should end with a single newline charactermarkdownlintMD047

Reply:
> Is that to say that storage/container profiles are not DWS resources?

Correct, they are NNF resources. Test Options should be used for administrative resources. I added a section on Test Options to clarify.